scripts/fiber/tests-passing.txt

```diff
@@ -891,7 +894,6 @@ src/renderers/dom/stack/client/__tests__/ReactDOM-test.js
 * should allow children to be passed as an argument
 * should overwrite props.children with children argument
 * should purge the DOM cache when removing nodes
-* allow React.DOM factories to be called without warnings
```

Comment:
Did some furniture moving of the factory tests, which failed CI. So I updated the test recording using:

```
scripts/fiber/record-tests --track-facts
```

Did I do this correctly?